caom2utils/caom2utils/parsers.py

```diff
@@ -1879,15 +1885,28 @@ def _get_num_parts(self):
             result = 1
         return result
 
+    def _set_wcs_parsers(self, obs_blueprint):
+        # self._wcs_parsers[0] = WcsParser(obs_blueprint, extension=0)
```

Comment:
commented on purpose?

Reply:
Yes. The commented out bit is the implementation that's being over-ridden. The commented-out bit has been removed.